src/Repositories/MovieRepository.cs

```diff
@@ -0,0 +1,19 @@
+using System.Linq;
+using MvcMovie.Repositories.Interfaces;
+using MvcMovie.Repositories.Database;
+using MvcMovie.Models.Database;
+
+namespace MvcMovie.Repositories
+{
+    public class MovieRepository : Repository<Movie>, IMovieRepository
+    {
+        public MovieRepository(DataBaseContext context) : base(context)
+        {
+        }
+
+        public DataBaseContext DataBaseContext
```

Comment:
I think this attribute shouldn't be here. The goal of the Unit of Work and Repository patterns is to avoid using directly the Context. (In some situations is unavoidable, but not this case).